yarn.lock

```diff
@@ -20460,7 +20460,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"serve-static@npm:1.15.0":
+"serve-static@npm:1.15.0, serve-static@npm:^1.14.1":
```

Comment:
Did something is this PR cause this to change?

Reply:
Looks like it was introduced during:
https://github.com/bbc/simorgh/pull/10376/files
https://github.com/bbc/simorgh/pull/10378/files
https://github.com/bbc/simorgh/pull/10379/files